codeReview.js

```diff
@@ -0,0 +1,78 @@
+import { Router, Request, Response } from 'api-framework'; // express-like package
+ 
+   import { getDBConnection } from '../config/db';
+   import { User } from '../types/user';
+   import { sendEmail } from '../util/send_email';
+   import { sendText } from '../util/send_text';
+ 
+   const router = Router();
+   const db = getDBConnection();
+ 
+   const fetchUsersByEmail = async (emails: string[]) => {
+     const users: User[] = await db<User>('users')
+       .select('id', 'email', 'first_name', 'enabled_features')
+       .whereIn('email', emails);
+ 
+     if (!users) {
+       throw new Error('No users found');
+     }
+ 
+     return users;
+   };
+ 
+   router.patch('/features', async (req: Request, res: Response) => {
+     const { emails, feature } = req.body; // these should be a list of emails and a feature, respectively
+ 
+     const users = fetchUsersByEmail(emails);
+ 
+     const fetchedEmails = users.map((u) => u.email);
+     emails.forEach((email: string) => {
+       if (!fetchedEmails.find((e) => e === email)) {
+         return res.status(200).send({
```

Comment:
- consider using a different response code, only return a 200 if you have specific reason
- return in a forEach will not stop the loop, it will send a message for every missing email, if you want to stop a forEach, you must throw an exception, or switch to a normal for loop
- res.send called multiple times, because its in a loop